.gitmodules

```diff
@@ -40,3 +40,6 @@
 [submodule "_includes/dot-github"]
 	path = _includes/dot-github
 	url = https://github.com/json-schema-org/.github.git
+[submodule "path/to/submodule"]
```

Comment:
We should not import this data as submodule. We like to avoid this type of dependency from projects outside our org. We'll do our own version of the data file in our data folder.